kernelci.org

```diff
@@ -92,11 +92,11 @@ cmd_rootfs_update() {
     cd checkout/kernelci-core
     git fetch origin main
     git checkout $FETCH_HEAD
-    sed -i s/"[0-9]\{8\}\.[0-9]\(\/{arch}\)"/"$version\1"/g config/core/test-configs.yaml
-    if ./kci_test validate_rootfs_urls --verbose; then
+    if ./kci_test update_rootfs_urls --release ${version} --output rootfs-images-${version}.yaml --verbose | grep OK; then
```

Comment:
Another way to do this is with `--output config/core/rootfs-images.yaml` and then if `git diff` shows nothing then there's no git commit to push.

Otherwise, with this current implementation the `rootfs-images-${version}.yaml` would need to be deleted in an `else` branch.

Reply:
Also I think the command will return an error if it fails, so that would be a case to take into account when updating the file in-place.

Reply:
One issue I can think of is relying on `OK` to be in the messages might be a bit fragile, as that's just to inform the user in verbose mode that some URLs are getting updated.  If the log messages changed, it would break in a rather subtle way.